src/behavior.scm

```diff
@@ -291,17 +291,17 @@
 			)
 			;; Were we interacting with someone else?  If so, then
 			;; maybe glance at the location of the person who left.
-			(SequentialAnd
-				(DefinedPredicate "Is interacting with someone?")
-				(SequentialOr
-					(NotLink (DefinedPredicate "dice-roll: glance lost face"))
-					(FalseLink (DefinedSchema "glance at lost face"))
-					(Evaluation (GroundedPredicate "scm: print-msg")
-						(ListLink (Node "--- Glance at lost face"))))
-				(TrueLink)
-			)
-			(Evaluation (GroundedPredicate "scm: print-msg")
-				(ListLink (Node "--- Ignoring lost face")))
+			;(SequentialAnd
+			;	(DefinedPredicate "Is interacting with someone?")
+			;	(SequentialOr
+			;		(NotLink (DefinedPredicate "dice-roll: glance lost face"))
+			;		(FalseLink (DefinedSchema "glance at lost face"))
+			;		(Evaluation (GroundedPredicate "scm: print-msg")
+			;			(ListLink (Node "--- Glance at lost face"))))
+			;	(TrueLink)
+			;)
+			;(Evaluation (GroundedPredicate "scm: print-msg")
+			;	(ListLink (Node "--- Ignoring lost face")))
 			(TrueLink)
```

Comment:
Why are you removing this code?  Can you provide some explanation? Did something break?

According to the comments in the code, this clause is performing some actual, useful work. Now it won't work any more, and there is no explanation for why or why not.

Please have other people review your code before committing changes.